frontend/packages/kubevirt-plugin/src/components/table/validation-cell.scss

```
@@ -4,3 +4,14 @@
 .kubevirt-validation-cell__cell--warning {
```

```suggestion
.kv-validation-cell__cell--warning {
```

can we unite these as well?